pyproject.toml

```diff
@@ -121,19 +121,23 @@ extend-exclude = '''
 
 
 [tool.ruff]
-exclude = [
-  "*/__init__.py"
-]
 
 [tool.ruff.lint]
-select = ["F401", "I"]  # TODO: eventually, expand to other 'F' linting
+select = ["F401", "I", "D101"] # TODO: eventually, expand to other 'F' linting
+ignore = ["D100"]
 fixable = ["ALL"]
 
+[tool.ruff.lint.per-file-ignores]
+"**__init__.py" = ["F401", "I"]
+"tests/**" = ["D"]  # We are not enforcing docstrings in tests
+"src/neuroconv/tools/testing/data_interface_mixins.py" = ["D"]  # We are not enforcing docstrings in tests
+
 [tool.ruff.lint.isort]
 relative-imports-order = "closest-to-furthest"
 known-first-party = ["neuroconv"]
 
-
+[tool.ruff.lint.pydocstyle]
+convention = "numpy"
```

Comment:
I think this is true here as well and this line is superfluous here.  I think I added it for testing to see if it was doing something different and saw no effect.

Reply:
Removed in the last commit.

Reply:
I think it is better to pick them as we like instead of by bundle.

Reply:
> I think it is better to pick them as we like instead of by bundle.

Seems reasonable to me.  In that case, we probably don't need `ignore = ["D100"]` either, since it hasn't been selected.

Reply:
I think you are right, let me check.

Reply:
Removed.